src/pages/ServiceDetails/ServiceMetrics.tsx

```diff
@@ -160,10 +71,11 @@ class ServiceMetrics extends React.Component<ServiceId, ServiceMetricsState> {
           requestDurationOut: metrics.histograms['request_duration_out'],
           responseSizeIn: metrics.histograms['response_size_in'],
           responseSizeOut: metrics.histograms['response_size_out'],
-          health: health
+          health: metrics.health
         });
       })
       .catch(error => {
+        // TODO: show error alert
```

Comment:
I guess alert handling won't be part of the current PR ? am I right ?
(I am fine with it, just checking before to merge).